### PR TITLE
fix(chat): Fix approve section incorrect name after dote (Issue #1425)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Folders' names should be displayed in the same way as it was in original publish request

Issues:

- Issue #1425 

**UI changes**

Before:
Снимок экрана 2024-06-04 в 17 50 31

After:
Снимок экрана 2024-06-04 в 17 53 31

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1496.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1496.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/Publish/PublishWizard.tsx [hunk 1]
```diff
@@ -217,7 +217,7 @@ export function PublishModal({
             }))
           : otherTargetAudienceFilters;
       const folderRegExp = new RegExp(
-        entity.folderId.split('/').slice(2).join('/'),
+        entity.folderId.split('/').slice(1).join('/'),
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/Publish/PublishWizard.tsx]: Why do we remove only 1 first path component?
image
And how it's related to dots in the end of name?

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1496.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/Publish/PublishWizard.tsx (hunk 1 above)
Comment (Derikyan) [reply apps/chat/src/components/Chat/Publish/PublishWizard.tsx]: I corrected the mistake, it was in a different part

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**